addon_toolkit/interfaces/citation.py

```diff
@@ -31,7 +31,7 @@ class CitationConfig:
     external_account_id: str | None = None
 
 
-class ItemType(Enum):
+class ItemType(StrEnum):
```

Comment:
Same here

Reply:
This fixes filters for `get_root_items` and `list_child_items`